pymc_marketing/mmm/utils.py

```diff
@@ -387,3 +388,32 @@ def create_new_spend_data(
             spend,
         ]
     )
+
+
+def _get_distribution(dist: dict) -> Callable:
```

Comment:
can we have a more descriptive name such as ` _get_distribution_from_dict` since ` _get_distribution` reads a bit too generic (just a suggestion)